react-toggle-switch-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import './App.css';
+
+export default function App() {
+  const [enabled, setEnabled] = useState(false);
+  const modifiedText = enabled ? 'on' : 'off';
+  const fontColor = {
+    color: modifiedText === 'on' ? 'white' : 'black',
+  };
+
+  function handleClick() {
+    document.body.style.backgroundColor = enabled ? 'white' : '#181818';
```

Comment:
You should be able to just use the css classes `line 19` to change your background color in `App.css`